exchanges/bitstamp/bitstamp_websocket.go

```diff
@@ -264,3 +247,48 @@ func (b *Bitstamp) WsUpdateOrderbook(ob PusherOrderbook, p currency.Pair, assetT
 
 	return nil
 }
+
+func (b *Bitstamp) seedOrderBook() error {
+	p := b.GetEnabledCurrencies()
+	for x := range p {
+		orderbookSeed, err := b.GetOrderbook(p[x].String())
+		if err != nil {
+			return err
+		}
+
+		var newOrderBook orderbook.Base
+		var asks []orderbook.Item
```

Comment:
can be rewritten: 
```go 
var asks, bids []orderbook.Item
```

Reply:
I am looking at moving over to using websocket to load the snapshot for the orderbook instead of rest so this may change completely